xdai.json

```diff
@@ -294,6 +294,7 @@
         },
         {
             "address": "0xB25F0eE2d26461E2B5B3D3DDAFE197A0dA677B98",
+            "unipool": "0xD93d3bDBa18ebcB3317a57119ea44ed2Cf41C2F2",
```

Comment:
@sembrestels shouldn't the unipool come from the subgraph? why we added it here?

Reply:
No, because Giveth is using a special kind of Unipool that we are not keeping track of on the subgraph.